config.json

```diff
@@ -554,7 +585,16 @@
         "slug": "robot-simulator",
         "name": "Robot Simulator",
         "uuid": "e5e55560-852f-4551-b4da-c9f4a3141470",
-        "prerequisites": [],
+        "prerequisites": [
+          "multiple-clause-functions",
+          "guards",
+          "pattern-matching",
+          "case",
+          "strings",
+          "enum",
+          "structs",
+          "errors"
```

Comment:
I am adding "errors" as a prerequisite to each exercise that uses the `{:ok, any} | {:error, any} | :error` pattern. I'm not entirely sure if that's a good idea because the "errors" concept is very far down the concept tree. Maybe it would be better to assume that tuples and atoms are enough? What do you think?

Reply:
I could go either way on this.  If there aren't enough exercises earlier on, we could leave that out an let tuples be enough. If there are, we could leave it and wait to name the pattern before the exercise appears.

Reply:
We have very few exercises early on. I think I'm going to use this concept as a way of steering people into easier exercises. So for easy ones, like `triangle`, I'm going to remove `errors` as a prerequisite so that people can do it early. They are unlikely to overcomplicate those by trying to raise and rescue :). But for complicated ones, like `forth` or `robot-simulator`, I'll keep `errors` as a prerequisite, "for the student's own good".